thoth/storages/graph/postgres.py

```diff
@@ -4195,6 +4197,117 @@ def get_adviser_run_count_per_source_type(
 
         return self._filter_source_type(results=results)
 
+    @staticmethod
+    def _create_date_filter(
+        date_: str
+    ) -> date:
+        """Create date filter.
+
+        @params date_: DD-MM-YY
+        """
+        day_month_year = date_.split('-')
+        date_filter = date(
```

Comment:
`datetime.strptime(date, '%DD-%MM-%YY').date()`
Not sure if this requires a function

Reply:
Could even parameterize the format and give a default

Reply:
Thanks Kevin, defintely better solution and we can catch error, so that user can find out if the format inserted is not correct!

Reply:
```

Traceback (most recent call last):
  File "test.py", line 8, in <module>
    print(len(graph.get_adviser_run_document_ids_all(initial_date="43-12-2020", count=None)))
  File "/home/fmurdaca/work/aicoe/storages/thoth/storages/graph/postgres.py", line 4232, in get_adviser_run_document_ids_all
    date_filter = self._create_date_filter(initial_date)
  File "/home/fmurdaca/work/aicoe/storages/thoth/storages/graph/postgres.py", line 4207, in _create_date_filter
    return datetime.strptime(date_, '%d-%m-%Y').date()
  File "/usr/lib64/python3.8/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib64/python3.8/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '43-12-2020' does not match format '%d-%m-%Y'

```